Sources/CacheAdvance/CacheAdvance.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  Created by Dan Federman on 11/9/19.
+//  Copyright © 2019 Dan Federman.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+/// A cache that enables the performant persistence of individual messages to disk.
+/// This cache is intended to be written from and appended to from the same serial queue.
```

Comment:
"written from and appended to from" -> "written and appended to from"?

Reply:
Changing to: `This cache is intended to be written to and read from the same serial queue.` "written from" and "appended to" are the same freaking action 🤦‍♂️. Thanks for calling this out.